src/template_parser.h

```diff
@@ -56,7 +56,7 @@ struct ParserTraits<char> : public ParserTraitsBase<>
 {
     static std::regex GetRoughTokenizer()
     {
-        return std::regex(R"((\{\{)|(\}\})|(\{%)|(%\})|(\{#)|(#\})|(\n))");
+        return std::regex(R"((\{\{)|(\}\})|(\{%[\+\-]?\s+raw\s+[\+\-]?%\})|(\{%[\+\-]?\s+endraw\s+[\+\-]?%\})|(\{%)|(%\})|(\{#)|(#\})|(\n))");
```

Comment:
I'm not sure we should do right trimming for `raw` and left trimming for `endraw`. Need to check original jinja implementation.
Also I think we can use UNIVERSAL_STR here and get rid of copy/paste of the regexp string.

Reply:
Let me now and I change that.

Reply:
I've just checked. We should do trimming. So the current implementation is correct.